tests/specs/tests.scss

```diff
@@ -0,0 +1,12 @@
+@import "true";
+
+// Import all scss files
+@import "include";
+
+// Import scss test files
+@import "scss/helpers/_convert-units.test";
```

Comment:
Imported partial `scss/helpers/_convert-units.test` should be written as `scss/helpers/convert-units.test`